src/modules/sub-items/components/table-view/table.view.component.js

```diff
@@ -18,140 +20,42 @@ export default class TableViewComponent extends Component {
         this.sortByDate = this.sortByDate.bind(this);
         this.sortByPriority = this.sortByPriority.bind(this);
         this.renderItem = this.renderItem.bind(this);
-        this.onSelectAll = this.onSelectAll.bind(this);
-
-        this.state = {
-            items: props.items,
-            isAscSort: false,
-            sortKey: false,
-        };
-    }
-
-    UNSAFE_componentWillReceiveProps({ items }) {
-        this.sortItems(items);
+        this.selectAll = this.selectAll.bind(this);
     }
 
     /**
-     * Sorts provided items
-     *
-     * @method sortItems
-     * @param {Array} items
-     * @memberof TableViewComponent
+     * Changes sort to name
      */
-    sortItems(items) {
-        const sortKey = this.state.sortKey;
-
-        if (!sortKey) {
-            this.setState(() => ({ items }));
-        }
+    sortByName() {
+        const { onSortChange } = this.props;
```

Comment:
Where is this method/function defined?

Reply:
in `sub.items.module.js`, it is past via props here: https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/112/files#diff-bebc1f75bde44101c137d4bc83684925R872